app/models/content_configuration.rb

```diff
@@ -17,19 +17,19 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
   has_attached_file :home_hero, default_url: "/assets/home/home.jpg"
 
   # Producer sign-up page
-  preference :producer_signup_pricing_table_html, :text, default: "(TODO: Pricing table)"
-  preference :producer_signup_case_studies_html, :text, default: "(TODO: Case studies)"
-  preference :producer_signup_detail_html, :text, default: "(TODO: Detail)"
+  preference :producer_signup_pricing_table_html, :text, default: I18n.t(:content_configuration_pricing_table)
```

Comment:
Should we introduce `models.content_configuration.pricing_table`? Or just `content_configuration.pricing_table`?